src/machine/machine_rp2040_gpio.go

```diff
@@ -22,7 +22,7 @@ type irqCtrl struct {
 }
 
 type ioBank0Type struct {
-	io                 [30]io
+	io                 [30]rpio
```

Comment:
Was this change intentional?

Reply:
Importing the `io` package as part of the type check causes a compile error in the smoke test for RP2040 because of it has an internal type called `io` as well.  The alternative would have been to import `io` with an alias, but I thought it would better to rename the type for the rp2040 to prevent this problem from happening again in the future.